### ISSUE TITLE
V1.0.18: Logging flooded with websocket errors

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#156]
If I cast to a single CC I only have these errors sporadically. While casting to a Group of 3 CC's the log is flooded with these errors. This immediately stops when pausing the music

```
547493887328] Client exceeded max pending messages [2]: 2048
17:53:27 – (ERROR) Home Assistant WebSocket API - message first occurred at 17:46:51 and shows up 1065 times
[547493887328] Client unable to keep up with pending messages. Stayed over 512 for 5 seconds
17:53:26 – (ERROR) Home Assistant WebSocket API - message first occurred at 17:46:53 and shows up 67 times

```

Note: this logging is for a few minutes. Release notes for v1.0.18 don't explain this. Related to chromecast updates in core-2022.5.4?!

**Comment by erkr** [comment]
Update: I see these loggings more than ever since the last Core-2022.5.4 update. Also when MA is not active they also intermittently pop up several times per hour. When casting in MA the logs are continuously flooded (multiple times per second).
It seems MA triggers a performance issue related to the lastest Core 2022.5.4 update

**Comment by marcelveldt** [comment]
I did not yet upgrade my production/family instance to 2022.5.4. That one is now without errors.
I'll take a look if these issues appear when I upgraded. Sounds to me like a bug or some endless loop.

**Comment by marcelveldt** [comment]
I couldn't reproduce this on both my production and dev setup. I even left my dev machine playing overnight in the debugger to trace all activity. I think this issue is either caused by something else or something on your setup I didn't account for.

Did you now add the template players to MA or the original cast players ?

**Comment by erkr** [comment]
Update: Specifically with version v1.0.18,  I  sporadically observed these loggings, even if I wasn't casting, As I updates to v1.0.18 and core-2022.5.4. is one go, it was not clear what was the cause. 
Yesterday I updated to v1.1.0 and v1.1.1 but had no opportunity to test (the family was looking at the song festival ;-).
Since I updated to V1.1.0 and V1.1.1 there were no sporadic errors anymore in the logging (but not casting)
Now I'm testing v1.1.2 and after casting to the group for one hour didn't cause any errors!!!

> Did you now add the template players to MA or the original cast players ?

My setup is still using Google cast entities as inputs and the mass players used in the universal templates players for control. As explained in the power toggel feature proposal, I foresee a power off state issue; 
My understanding of these states:
- a CC in off state is available, but just not connected/used by any app
- The template in off state is really off, so the embedded player is unavailable, while the template state is off
